lib/helpers/document/getDeepestSubdocumentForPath.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+module.exports = function getDeepestSubdocumentForPath(doc, parts, schema) {
```

Comment:
it would probably help to add jsdoc for this, so it can be understood more easily at a glance